irteus/irtrobot.l

```diff
@@ -826,6 +826,41 @@
                             (setq dx (+ dx ddx) dy (+ dy ddy) dth (+ dth ddth)))))
        (push (funcall gen-go-pos-step-node-func mc leg leg-translate-pos) ret)
        (reverse ret))))
+  (:go-pos-quadruped-params->footstep-list
+   (xx yy th ;; [mm] [mm] [deg]
+       &key (type :crawl))
+   "Calculate foot step list for quadruped walking from goal x position [mm], goal y position [mm], and goal yaw orientation [deg]."
+   (let ((biped-fsl (send *robot* :go-pos-params->footstep-list xx yy th :forward-offset-length 100 :rotate-rad 3))
```

Comment:
`*robot*` should be `self`.

Please try the following command to check global variable:
```
cd ~/ros/hydro_parent/src/jskeus/irteus
make
```